libs/@local/hashql/core/src/module/locals.rs

```diff
@@ -94,15 +94,11 @@ pub struct Local<'heap, T> {
     pub value: T,
 }
 
-impl<'heap, T> PrettyPrint<'heap> for Local<'heap, T>
+impl<'heap, E, T> PrettyPrint<'heap, E> for Local<'heap, T>
```

Comment:
So we agreed on having more than single-letter lifetimes. This is not really according to the common coding guidelines of Rust, but we all agreed that it makes things easier. What is the take on single-letter generics? 
Pro: It's clear that if you see a single letter that it's a generic
Con: `T` or `E` could be anything. 

I don't feel strongly but am interested on your view.